packages/manager/modules/vrack/src/dashboard/vrack.controller.js

```diff
@@ -171,24 +172,15 @@ export default class VrackMoveDialogCtrl {
         });
     }
 
-    this.openModalAddSubnet = (confirmHandler) => {
-      this.addSubnetModalConfirm = (subnet, adress) => {
-        confirmHandler(subnet, adress);
-      };
-
-      this.isOpenModalAddSubnet = true;
-    };
-
-    this.subnetUpdatedFalback = () => {
-      this.resetCache();
-      this.refreshData();
-    };
-
     this.$scope.$on('$destroy', () => {
       if (this.poller) {
         this.$timeout.cancel(this.poller);
       }
     });
+
+    this.isIPv6 = (service) => {
+      return !!service.id.match(/::/);
+    };
```

Comment:
try to keep on way for declaring function

```suggestion
     static isIPv6(service) {
      return !!service.id.match(/::/);
    };
```